freqtrade/plot/plotting.py

```diff
@@ -33,21 +33,24 @@ def init_plotscript(config):
         pairs = config["pairs"]
     else:
         pairs = config["exchange"]["pair_whitelist"]
+    assert pairs is not None, ('No pairs available in config.')
```

Comment:
The checks you add here with `assert` do make sense.

However, please don't use assert statements in running code (use if / `raise OperationalException(message)` instead). 

I know this is sometimes suggested for analytic workflows (maybe also elsewhere?) - but python can be run in an "optimized" mode (`python -O`) where asserts are not evaluated / skipped.

It's not a problem everyone will run into - but some might (we have nothing that would prevent this mode at the moment, and i'd like to keep it that way).

Reply:
I replaced the `assert` statements with `OperationalException`

Reply:
did you forget to push ...? (or pushed to the wrong branch?) as u i don't seem to see these changes...